global.json

```diff
@@ -2,7 +2,7 @@
   "sdk": {
     "version": "8.0.100-preview.7.23376.3",
     "allowPrerelease": false,
-    "rollForward": "latestPatch"
+    "rollForward": "disable"
```

Comment:
is this change intentional?

Reply:
yes.  @jasonmalinowski for more context.  but this unblocks people being able to actually develop with roslyn locally :)